api/src/context.rs

```diff
@@ -43,9 +43,10 @@ use aptos_types::{
     },
     transaction::{SignedTransaction, TransactionWithProof, Version},
 };
+use aptos_utils::aptos_try;
 use aptos_vm::{
     data_cache::{AsMoveResolver, StorageAdapter},
-    move_vm_ext::MoveResolverExt,
+    move_vm_ext::AptosMoveResolver,
```

Comment:
Is `AptosMoveResolver` just a renaming for `MoveResolverExt`? Or are they different in functionality?